bin/check-azurerm-monitor-metric.rb

```diff
@@ -62,7 +62,14 @@ class CheckAzurermMonitorMetric < Sensu::Plugin::Check::CLI
          description: 'Port used to authenticate when using the local identity via Managed Service Identity (MSI)',
          short: '-o PORT',
          long: '--local-auth-port PORT',
-         default: '50342'
+         proc: proc(&:to_i),
+         default: 50_342
+
+  option :assigned_identity_resource,
```

Comment:
other than the trailing `/` this looks the same as `base_url` would they ever be different? If not lets use one option and where we need the slash it can be part of the implementation itself. Maybe this can be used against an internal AD or something but I don't know much about AD and Azure to be honest.

Reply:
I don't know if they will always be the same.  Plus one is used all the time, and one is used some of the time, so i thought it would be clearer to be 2 separate options.

Reply:
ok just wanted to see if there was a valid reason and if so I am :+1: